donut/modules/groups/utils/position_importer.py

```diff
@@ -0,0 +1,100 @@
+import argparse
+from donut.pymysql_connection import make_db
+import csv
+
+
+def import_file(env, filename):
+    # Create reference to DB
+    db = make_db(env)
+    try:
+        db.begin()
+        with open(filename, "r") as f:
+            reader = csv.DictReader(f, delimiter=",")
+            for i, line in enumerate(reader):
+                uid = line['uid']
+                group_name = clean_names(line['org_name'])
+                pos_name = line['pos_name']
+                control = line['control']
+                start_date = line['start_date']
+                end_date = line['end_date']
+                group_id = get_create_group_id(group_name, db)
+                pos_id = get_create_position_id(pos_name, group_id, control,
+                                                db)
+                user_id = get_user_id_from_uid(uid, db)
+
+                query = """
+                INSERT INTO position_holders(pos_id, user_id, start_date, end_date) VALUES(%s, %s, %s, %s) 
+                """
+                with db.cursor() as cursor:
+                    cursor.execute(query,
+                                   [pos_id, user_id, start_date, end_date])
+        db.commit()
+    finally:
+        db.close()
+
+
+def get_create_group_id(group_name, db):
+    # I think the vast majority of non-auto created groups are committees.
+    insertion_query = "INSERT INTO groups(group_name, type) VALUES(%s, 'committee')"
+    res = get_group_id(group_name, db)
+    if res is None:
+        with db.cursor() as cursor:
+            cursor.execute(insertion_query, [group_name])
+        res = get_group_id(group_name, db)
+    return res['group_id']
+
+
+def get_group_id(group_name, db):
+    group_id_query = "SELECT group_id FROM groups WHERE group_name=%s"
+    with db.cursor() as cursor:
+        cursor.execute(group_id_query, [group_name])
+        res = cursor.fetchone()
+    return res
+
+
+def clean_names(group_name):
+    return group_name.replace("Hovse", "House")
```

Comment:
lol

Reply:
House group names should just be the house name (without `House` or `Hovse`), sorry. I fixed this in #135.